src/core-services/shop/queries/index.js

```diff
@@ -1,18 +1,9 @@
-/* eslint-disable node/no-deprecated-api */
-/* TODO: revisit `url.parse` throughout Reaction */
-import url from "url";
```

Comment:
There are a couple instances where you are removing `url`, which I've also done in this PR here:

Just making a note to check for conflicts whichever is merged first. https://github.com/reactioncommerce/reaction/pull/5717